examples/graphql-kotlin/src/test/kotlin/example/armeria/server/graphql/kotlin/GraphqlArmeriaClient.kt

```diff
@@ -0,0 +1,53 @@
+package example.armeria.server.graphql.kotlin
+
+import com.expediagroup.graphql.client.GraphQLClient
+import com.expediagroup.graphql.client.serializer.GraphQLClientSerializer
+import com.expediagroup.graphql.client.serializer.defaultGraphQLSerializer
+import com.expediagroup.graphql.client.types.GraphQLClientRequest
+import com.expediagroup.graphql.client.types.GraphQLClientResponse
+import com.linecorp.armeria.client.WebClient
+import com.linecorp.armeria.common.HttpMethod
+import com.linecorp.armeria.common.HttpRequest
+import com.linecorp.armeria.common.HttpRequestBuilder
+import com.linecorp.armeria.common.MediaType
+import java.io.Closeable
+import java.net.URL
+
+class GraphqlArmeriaClient(
+    private val url: URL,
```

Comment:
It would be convenient to directly take a URI returned by `server.httpUri()`.
```suggestion
    private val uri: URI,
```